bridges/YoutubeBridge.php

```diff
@@ -110,8 +110,12 @@ private function ytBridgeParseXmlFeed($xml) {
 		$this->feedName = $this->ytBridgeFixTitle($xml->find('feed > title', 0)->plaintext);  // feedName will be used by getName()
 	}
 
-	private function ytBridgeParseHtmlListing($html, $element_selector, $title_selector){
-		$limit = 10;
+	private function ytCountItemsFromHtmlListing($html, $element_selector, $title_selector) {
+		return $this->ytBridgeParseHtmlListing($html, $element_selector, $title_selector, false);
+	}
```

Comment:
This function is a wrapper that adds a constant to another function call. Please keep it simple and directly call the inner function instead.